### PR TITLE
Enrich backbone.ribs's package.json(field: license)

### PR DESCRIPTION
**Pull request by iwdmb**
#5500 
license link: [https://github.com/ZaValera/backbone.ribs/blob/master/bower.json](https://github.com/ZaValera/backbone.ribs/blob/master/bower.json)

### CODE REVIEW

File: ajax/libs/backbone.ribs/package.json [hunk 1]
```diff
@@ -17,5 +17,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/ZaValera/backbone.ribs.git"
-  }
+  },
+  "license": "MIT"
 }
+
```
Comment (PeterDaveHello) [inline ajax/libs/backbone.ribs/package.json]: Should not add this empty line here.

**Comment by iwdmb** [comment]
fix

**Comment by PeterDaveHello** [comment]
Would you please rebase with our latest master branch? Thanks.

**Comment by iwdmb** [comment]
Please close this pull request. Thanks!

**Pull request closed without merging**